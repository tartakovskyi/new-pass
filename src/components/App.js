import React from 'react';
import { ChakraProvider, theme, Box, Heading, VStack } from '@chakra-ui/react';
import Form from './Form';

const App = () => {
  return (
    <ChakraProvider theme={theme}>
      <Box py="80px">
        <VStack direction="column" align="center">
          <Heading as="h1" textTransform="uppercase" mb={4}>
            Create Your New Password
          </Heading>
          <Form />
        </VStack>
      </Box>
    </ChakraProvider>
  );
};

export default App;
