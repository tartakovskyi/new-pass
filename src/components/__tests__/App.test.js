import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

test('App should render correctly', () => {
  const { container } = render(<App />);

  expect(container.firstChild).toMatchSnapshot();
});
