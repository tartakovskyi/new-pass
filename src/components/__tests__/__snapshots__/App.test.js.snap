// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  class="css-36s324"
>
  <div
    class="chakra-stack css-owjkmg"
  >
    <h1
      class="chakra-heading css-ad97t1"
    >
      Create Your New Password
    </h1>
    <form
      class="css-kx6wjg"
    >
      <div
        class="chakra-form-control css-c1qqtu"
        role="group"
      >
        <label
          class="chakra-form__label css-2gx1h6"
          for="oldPass"
          id="oldPass-label"
        >
          Old password
        </label>
        <input
          class="chakra-input css-1lw1oo1"
          id="oldPass"
          name="oldPass"
          type="password"
          value=""
        />
      </div>
      <div
        class="chakra-form-control css-c1qqtu"
        role="group"
      >
        <label
          class="chakra-form__label css-2gx1h6"
          for="newPass"
          id="newPass-label"
        >
          New password
        </label>
        <input
          class="chakra-input css-1lw1oo1"
          id="newPass"
          name="newPass"
          type="password"
          value=""
        />
      </div>
      <div
        class="chakra-form-control css-c1qqtu"
        role="group"
      >
        <label
          class="chakra-form__label css-2gx1h6"
          for="confirmPass"
          id="confirmPass-label"
        >
          Confirm new password
        </label>
        <input
          class="chakra-input css-1lw1oo1"
          id="confirmPass"
          name="confirmPass"
          type="password"
          value=""
        />
      </div>
      <button
        class="chakra-button css-1en85ns"
        disabled=""
        type="submit"
      >
        Save changes
      </button>
    </form>
  </div>
</div>
`;
